src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -1553,7 +1553,7 @@ private void ExecuteInstallPackageCommand(object sender, ExecutedRoutedEventArgs
         private void PackageList_UpdateButtonClicked(PackageItemListViewModel[] selectedPackages)
         {
             var packagesToUpdate = selectedPackages
-                .Select(package => new PackageIdentity(package.Id, package.LatestVersion))
+                .Select(package => new PackageIdentity(package.Id, package.Version))
```

Comment:
welp :D

Nice catch! 

What was the repro? 

Going from the latest to a lower version or? I recall this being a tough one to figure out.

Reply:
Not hard to find this part, I changed it in my last PR in this area. 
I don't 100% understand why, but only 'N' LatestVersions get populated. *If any of the remaining* packages have a prerelease version, *BOOM*.